docs/extend/add_metrics.md

```diff
@@ -1 +1,63 @@
-Under Construction
+# Add new metrics
+
+In this section we describe the process of adding new metrics in KOBE.
+New metrics can be added in a federator or in an evaluator.
```

Comment:
Fluentd captures logs from all containers running so it can also capture logs from e.g. a triple-store used in KOBE. So metrics can also be collected from other components than federators or evaluators.

Reply:
Ok, I will remove this sentence.